lib/model/population_criteria.rb

```diff
@@ -106,6 +107,8 @@ def get_logic_leaves(children=nil)
     def to_model
       mps = preconditions.collect {|p| p.to_model}
       HQMF::PopulationCriteria.new(id, hqmf_id, type, mps, title, aggregator)
+      p "BANANA"
+      p HQMF::PopulationCriteria.new(id, hqmf_id, type, mps, title, aggregator)
```

Comment:
Yes.. I removed them at one point but I reverted some changes at one point. I am removing them and pushing again.